unittests/savanna_cluster.hpp

```diff
@@ -127,6 +129,8 @@ namespace savanna_cluster {
 
       size_t& vote_delay() { return _vote_delay; }
 
+      void propagate_delayed_votes_to(node_t &to);
```

Comment:
`node_t& to`

Reply:
https://github.com/AntelopeIO/spring/pull/782/commits/41ceebcdd7364e21f84f613c872aa112d1845abe